src/main/java/com/github/jcustenborder/kafka/connect/spooldir/AbstractSourceConnectorConfig.java

```diff
@@ -40,6 +40,12 @@
   public static final String FILE_MINIMUM_AGE_MS_CONF = "file.minimum.age.ms";
   public static final String FILE_SORT_ATTRIBUTES_CONF = "files.sort.attributes";
 
+  public static final String INPUT_PATH_WALK_RECURSIVELY = "input.path.walk.recursively";
+  static final String INPUT_PATH_WALK_RECURSIVELY_DOC = "If enabled, activates a recursive walk of sub-directories under input.path searching for files matching input.file.pattern";
+  
+  public static final String INPUT_PATH_WALK_RECURSIVELY_RETAIN_SUB_DIRS = "input.path.walk.recursively.retain.sub.dirs";
+  static final String INPUT_PATH_WALK_RECURSIVELY_RETAIN_SUB_DIRS_DOC = "When input.path.walk.recursively is enabled, if discovered sub dirs under input.path will be retained regardless of success/failure of file processing";
```

Comment:
Can we update this to something like `cleanup.policy.maintain.relative.path`? This seems to be a behavior specific to the cleanup policy

Reply:
We could but I think its better keeping the configuration name as-is because it inherently relates it to the other configuration just by looking at it.

Reply:
ill just change this dont care too much honestly just need the functionality

Reply:
@jcustenborder this is checked in